### PR TITLE
Dependency refactor cleanup

### PR DESCRIPTION
**Pull request by joel-rieke**

### CODE REVIEW

File: atrium/vestibulum/trcdb/trcplgtoolbase/trcplgtoolbase.go [hunk 1]
```diff
@@ -327,7 +335,7 @@ func CommonMain(envPtr *string,
 	}
 
 	// Get existing configs if they exist...
-	pluginToolConfig, plcErr := trcvutils.GetPluginToolConfig(driverConfigBase, mod, coreopts.BuildOptions.ProcessDeployPluginEnvConfig(map[string]interface{}{}), *defineServicePtr)
```
Comment (joel-rieke) [inline atrium/vestibulum/trcdb/trcplgtoolbase/trcplgtoolbase.go:330]: Should be driverconfigbase.

File: pkg/capauth/creds.go [hunk 2]
```diff
@@ -18,10 +18,10 @@ import (
 )
 
 const (
-	ServCert           = "/etc/opt/vault/certs/serv_cert.pem"
```
Comment (joel-rieke) [inline pkg/capauth/creds.go:21]: revert these changes.

Comment (joel-rieke) [reply pkg/capauth/creds.go]: Fixed

**Review by KarnveerSGill — APPROVED** [review]
